src/ubuntu/20.04/helix/wasm/amd64/Dockerfile

```diff
@@ -11,18 +11,30 @@ RUN apt-get clean && \
     mv /etc/apt/sources.list1 /etc/apt/sources.list &&  apt-get update && \
     apt-get install -qq -y \
         libnode-dev \
+        curl \
         node-gyp \
         npm \
      && rm -rf /var/lib/apt/lists/* \
      \
      && localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
 
 
+USER helixbot
+
+# update node, which is required for newer npm, jsvu and v8
+RUN cd ~ && \
+    curl -LO https://nodejs.org/dist/v18.12.1/node-v18.12.1-linux-x64.tar.xz && \
```

Comment:
For convenience, it might be better to define an `ENV` variable for the Node version so it can be updated in one location. Currently, there's 3 different spots that need to be updated if the version changes.